src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+    city: 'Honolulu',
+    lat: 21.3099,
+    long: 157.8581,
+    temp: 72,
+};
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const findLatitudeAndLongitude = () => {
+    let location = state.city;
+    axios.get('http://localhost:5000/location',
+    {
+        params: {
+            q: location,
+        },
+    })
+    .then( (response) => {
+        console.log(response);
+        state.lat = response.data[0].lat;
+        state.long = response.data[0].lon;
+        console.log(`response: ${state.city}`);
+        console.log('success in findLatitudeAndLongitude!', state.lat, state.long);
+        
+        getWeather();
+    })
+    .catch( (error) => {
+    console.log(`error: ${state.city}`);
+    console.log('error in findLatitudeAndLongitude!');
+    console.log(error);
+    });
+};
+
+const getWeather = () => {
+    axios
+      .get('http://localhost:5000/weather', {
+        params: {
+          lat: state.lat,
+          lon: state.long,
+        },
+      })
+      .then((response) => {
+        const weather = response.data;
+        state.temp = Math.round(convertKtoF(weather.main.temp));
+        formatTempAndGarden();
+      })
+      .catch((error) => {
+        console.log('Error getting the weather:', error);
+      });
+};
+
+// --------------------------------------------------------------------------------------
+
+const updateSky = () => {
+    let sky = document.getElementById('sky');
+    let skyOption = document.getElementById('sky-select').value;
+    if (skyOption === 'Sunny') {
+        sky.textContent = "☁️☁️✨🌚☁️🌝☁️✨☁️";
+    } else if (skyOption === 'Cloudy') {
+        sky.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (skyOption === 'Rainy') {
+        sky.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else if (skyOption === 'Snowy') {
+        sky.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    }
+};
+
+const formatTempAndGarden = () => {
+    let temp = state.temp;
+    let color = 'blue';
+    let landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+
+    if (temp >= 80) {
+        color = 'red';
+        landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (temp >= 70) {
+        color = 'orange';
+        landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (temp >= 60) {
+        color = 'yellow';
+        landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (temp >= 50) {
+        color = 'green';
+    } else {
+        color = 'blue';
+        landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
```

Comment:
Another way to format this method would be to have an object that has temp values as keys and then you could have values be a list and the list could hold background color and background image, like:

```javascript
const tempDisplayObject = {80: ['red',  "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"], 70: ['orange', "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"]} 
```

When you have an object, you don't need to do if/else if/else if/ else which can introduce bugs and also adds complexity to the method. If you have an object, than you can iterate over the object and check the keys with state.currentTemp and then get the correct values to set the color and landscape emojis.